File: week1/day3/algo-deaf-grandma/deaf_grandma.js
function deafGrandma() {

    let goodbye_count = 0;

    console.log('HEY KID!');

    while (goodbye_count < 2) {
        let usr_input = window.prompt();

        if (usr_input === '') {
            console.log('WHAT?!')
        } else if (usr_input === 'GOODBYE!' && goodbye_count === 0) {
            console.log('LEAVING SO SOON?')
            goodbye_count = goodbye_count + 1
        } else if (usr_input === 'GOODBYE!' && goodbye_count === 1) {
            console.log('LATER, SKATER')
            break
        } else if (usr_input.toUpperCase === usr_input && usr_input != 'GOODBYE!') {
            console.log('NO, NOT SINCE 1946!')
        } else {
            console.log('SPEAK UP, KID')
        }

    }

}

deafGrandma()

File: week1/day3/algo-deaf-grandma/deaf_grandma.py
from curses.ascii import isupper


def deaf_grandma():

    goodbye_count = 0

    print('HEY KID!')

    while goodbye_count < 2:
        usr_input = input()

        if usr_input == '':
            print('WHAT?!')

        elif usr_input == 'GOODBYE!' and goodbye_count == 0:
            print('LEAVING SO SOON?')
            goodbye_count += 1

        elif usr_input == 'GOODBYE!' and goodbye_count == 1:
            print('LATER, SKATER!')
            break

        elif usr_input.isupper() and usr_input != 'GOODBYE!':
            print('NO, NOT SINCE 1946!')
        
        else:
            print('SPEAK UP, KID')


deaf_grandma()